trustfall_core/src/graphql_query/error.rs

```diff
@@ -24,6 +24,12 @@ pub enum ParseError {
     #[error("Directive {0} received value of inappropriate type for argument {1}")]
     InappropriateTypeForDirectiveArgument(String, String, Pos),
 
+    #[error(
+        "Directive @filter received a string instead of a list. \
```

Comment:
This doesn't really fit the style, a more fitting error message to the ones below would be `Directive @filter received a string, this is not supported.` However I feel that as this error also gives a possible solution, I think we should tell them what the right answer would be

Reply:
I think the new style is good here. Just one issue: "received" a string / list doesn't sound right because `@filter` takes two args, not one. Any way to tweak the wording to make this clearer?

Reply:
`Directive @filter's second argument is a string instead of a list.` OR
`Directive @filter's value argument is a string instead of a list.` (I prefer second)

Reply:
Perhaps `Value argument in @filter directive is a string instead of a list.` ?

Reply:
Yep, sounds great, I was sticking to the existing style but I'm good with this one too, I'll make the change.